.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,27 +6,39 @@ labels: bug
 
 ---
 
+**⚠️IMPORTANT⚠️ if you do not fill this out, we cannot help you.**
+
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**Plaintext**
+**⚠️IMPORTANT⚠️ The below code is non-negotiable for "Ciphey didn't decrypt...." problems. If you do not tell us your plainttext, we will not help you.** 
+
+```
+Include your plaintext here, replacing this 
+```
+
+**Verbose Output**
+**⚠️IMPORTANT⚠️ Verbose outputt will tell us why it's not working the way we expected it to be.**
```

Comment:
I like putting this near the top, but could we maybe ask for a pastebin/gist so that the report doesn't become unreadable? Also `s/outputt/output/`